frontend/assets/l10n/app_de.json

```diff
@@ -96,7 +96,8 @@
     "cameraAccessRequiredSettings": "Um einen QR-Code einzuscannen, benötigt die App Zugriff auf die Kamera. In den Einstellungen können Sie der App den Zugriff auf die Kamera erlauben.",
     "cardAlreadyActivated": "Der eingescannte QRCode wurde bereits aktiviert.",
     "cardExpired": "Ihre Karte ist abgelaufen. Unter \"Weitere Aktionen\" können Sie einen Antrag auf Verlängerung stellen.",
-    "cardInvalid": "Ihre Karte ist ungültig. Sie wurde entweder widerrufen oder auf einem anderen Gerät aktiviert.",
+    "cardExpiredBeforeActivation": "Ihre Karte ist bereits abgelaufen und kann nicht mehr aktiviert werden. Bitte beantragen Sie sie erneut.",
```

Comment:
it seems this message is never displayed.
deep link activation - card expired:
<img width="362" height="818" alt="image" src="https://github.com/user-attachments/assets/a9231644-f095-4c5f-82af-e8ca6b162f83" />
code scanning - card expired:
<img width="360" height="818" alt="image" src="https://github.com/user-attachments/assets/63c0cc03-7eec-4418-a810-fe1f63d571cb" />

Reply:
Not sure.. what do you suggest? so what I did is just a an extra check? 🙈
I think the qr expiration message is correct because its scanned code and not that really confusing.